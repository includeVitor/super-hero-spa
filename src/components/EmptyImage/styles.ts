import styled from 'styled-components'

const Article = styled.article`
  width: 14.875rem;
  height: 20rem;
  display: flex;
  align-items: center;
  justify-content: center;
  border: solid 0.125rem black;
  border-radius: 1rem;
`
export { Article }
