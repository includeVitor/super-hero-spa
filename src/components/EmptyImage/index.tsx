import { memo } from 'react'
import { Article } from './styles'

const EmptyImage = () => <Article>Image could not be loaded</Article>

export default memo(EmptyImage)
