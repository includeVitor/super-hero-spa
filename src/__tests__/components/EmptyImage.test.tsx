import { render } from '@testing-library/react'
import EmptyImage from '../../components/EmptyImage'

describe('EmptyImage', () => {
  it('should match the snapshot', () => {
    const { container } = render(<EmptyImage />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
