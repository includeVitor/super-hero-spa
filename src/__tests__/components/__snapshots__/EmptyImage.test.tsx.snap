// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyImage should match the snapshot 1`] = `
<article
  class="sc-bdvvtL gfFlDl"
>
  Image could not be loaded
</article>
`;
